.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,47 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
```

Comment:
Adding nodejs actions (Github CI) for simple build success/fail status on NodeJS 10 and NodeJS12